pkg/ebpf/bytecode/manager.go

```diff
@@ -0,0 +1,86 @@
+// +build linux_bpf
+
+package bytecode
+
+import (
+	"os"
+
+	"github.com/DataDog/ebpf/manager"
+)
+
+const (
+	// maxActive configures the maximum number of instances of the kretprobe-probed functions handled simultaneously.
+	// This value should be enough for typical workloads (e.g. some amount of processes blocked on the accept syscall).
+	maxActive = 128
```

Comment:
Should this be `min(128, runtime.NumCPU())` ?

Reply:
We've had multiple issues with `maxActive` on the security agent side. We'd recommend to keep a high value (our current `maxActive` is 512 for example) since `NumCPU` will not be enough for syscalls that can be pre-empted.

Also, after spending some (way too much) time on this, we also concluded that the second column in `kprobe_profile` is not the number of kretprobe misses (although we wouldn't be able to tell you what it is), and the only way to accurately keep track of the number of kretprobe misses is to calculate the difference between the number of kprobe and kretprobe hist for the same hook point.

Reply:
Note this wasn't a new value, just moved from `tracer.go`:
https://github.com/DataDog/datadog-agent/blob/bd0991d26ebcc5f02e3d25bad00bd96e292194fb/pkg/ebpf/tracer.go#L83-L85